recipe/meta.yaml

```diff
@@ -1,37 +1,119 @@
-{% set version = "5.5.2" %}
+{% set version = "5.7.0" %}
 {% set number = "0" %}
-{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '10.2' %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
 {% set build_ext_version = "1.0.0" %}
 {% set build_ext = "cuda" if cuda_enabled else "cpu" %}
-{% set build_string = "h{}_{}_{}".format(PKG_HASH, number, build_ext) %}
-{% set py_build_string = "py{}h{}_{}".format(CONDA_PY, PKG_HASH, number) %}
+{% set build_string = "h{}_{}".format(PKG_HASH, number) %}
+{% set build_string_ext = "{}_{}".format(build_string, build_ext) %}
+{% set py_build_string_ext = "py{}{}_{}".format(CONDA_PY, build_string, build_ext) %}
+{% set py_build_string = "py{}{}".format(CONDA_PY, build_string) %}
+{% set install_base = "opt/omnisci" %}
+{% set arrow_version = "2.*" %}
+{% set thrift_version = "0.13.*" %}
+# omniscidb 5.5 is not LLVM 12 ready
```

Comment:
this is for 5.7, so either comment has to be updated or constraint lifted :)

Reply:
LLVM 12 is not tested, it is not even landed in conda-forge. So I'll just update the comment.